plugins/webhooks/index.js

```diff
@@ -103,6 +104,9 @@ const webhooksPlugin = {
                     } catch (err) {
                         logger.error(`[${hookId}]: ${err}`);
 
+                        if (err.statusCode === 404) {
```

Comment:
it looks `err.status` and `err.statusCode` are mixed. Is this intended?

Reply:
It seems api returns both `status`, and `statusCode`. I will fix to use `statusCode`.
```
{
  status: 404,
  response: {
    url: 'xxxxxx'
    status: 404,
    headers: {
       **snip**
    },
    data: {
      message: 'Branch not found',
      documentation_url: 'https://docs.github.com/enterprise/3.2/rest/reference/repos#get-a-branch'
    }
  },
  request: {
    method: 'GET',
    url: 'xxxxxxx'
    headers: {
      accept: 'application/vnd.github.v3+json',
      'user-agent': 'octokit-rest.js/18.12.0 octokit-core.js/3.6.0 Node.js/12.22.11 (linux; x64)',
      authorization: 'token [REDACTED]'
    },
    request: { hook: [Function: bound bound register] }
  },
  statusCode: 404
}
```